### ISSUE TITLE
advise for the code

### ISSUE THREAD

**Issue by sanggehouzhihoujue** [eragonruan/refinenet#11]
The original code is written in matlab, and the network architectures are appended in the code package.
It's my honor to read your code in tensorflow, but there are some problems in your code different from the orignal network architectures. With respect, I list these codes in below pictures.
![qq 20180212104032](https://user-images.githubusercontent.com/31615736/36082063-3af7611e-0fe1-11e8-9874-06d998828308.png)
![qq 20180212104128](https://user-images.githubusercontent.com/31615736/36082171-cb698ac4-0fe1-11e8-925f-fe3f8d4d69cd.png)
![qq 20180212104215](https://user-images.githubusercontent.com/31615736/36082172-cb98c6b8-0fe1-11e8-8465-a1697a7ba6e9.png)
I will be waiting for your reply.

**Comment by sanggehouzhihoujue** [comment]
@eragonruan

**Comment by eragonruan** [comment]
@sanggehouzhihoujue thanks for the sharing. For the first part about resnet101, I got this model from [tensorflow/models](https://github.com/tensorflow/models/tree/master/research/slim/nets). And  I will re-check the second part soon.

**Comment by shipengai** [comment]
Hi,@sanggehouzhihoujue 
Did you test the model on PASCAL VOC validation set?
and train it on your own dataset?
Thanks.
I'm looking forward your reply

**Comment by sanggehouzhihoujue** [comment]
@shipeng-uestc  I trained it on the VOC-person part dataset, but the result is not so good. There should be some differences from the original code in matlab.

**Comment by shipengai** [comment]
Hi,@sanggehouzhihoujue .
Thank you very much for your reply.
Did you run the author's(@eragonruan ) demo.py? When I use the trained model from his link, some error about BN layer occurs.
'''NotFoundError (see above for traceback): Key feature_fusion/Conv_37/BatchNorm/beta/ExponentialMovingAverage not found in checkpoint'''

**Comment by sanggehouzhihoujue** [comment]
@shipeng-uestc  I ran the demo.py successfully. Do you create the dir named checkpoint?

**Comment by shipengai** [comment]
@sanggehouzhihoujue I think my ckpt has some error. Can you share me an old code or ckpt?
Thanks.

**Comment by sanggehouzhihoujue** [comment]
@shipeng-uestc  Didn't you train it on the VOC ? The ckpt is conducted at the training stage. The ckpt trained by  myself can't result in good performance. What's more, I only trained on the part dataset.
